kyo-bench/src/main/scala/kyo/bench/BroadFlatMapBench.scala

```diff
@@ -20,24 +20,13 @@ class BroadFlatMapBench extends Bench.SyncAndFork(BigInt(610)):
     def kyoBench() =
         import kyo.*
 
-        def kyoFib(n: Int): BigInt < IOs =
+        def kyoFib(n: Int): BigInt < IO =
             if n <= 1 then BigInt(n)
             else kyoFib(n - 1).flatMap(a => kyoFib(n - 2).flatMap(b => a + b))
 
         kyoFib(depth)
     end kyoBench
 
-    @Benchmark
```

Comment:
These `Kyo2` benchmarks were temporary to access the performance of the new design in `kyo-prelude`.